tests/builder/axes_test.py

```diff
@@ -347,8 +347,33 @@ def test_axis_mapping(ufo_module):
     assert doc.axes[1].map != wdth_mapping
     assert doc.axes[1].map == []
 
-    # When we convert back to glyphs, the wdth mapping isn't present because
-    # it was removed during the designspace conversion.
     font = to_glyphs(doc)
-    axis_mappings["wdth"] = {}
     assert font.customParameters["Axis Mappings"] == axis_mappings
+
+
+def test_axis_with_no_mapping_does_not_error_in_roundtrip(ufo_module):
+    from copy import deepcopy
+
+    doc = designspaceLib.DesignSpaceDocument()
+
+    # Add a "Regular" source
+    regular = doc.newSourceDescriptor()
+    regular.font = ufo_module.Font()
+    regular.location = {"Style": 0}
+    doc.addSource(regular)
+
+    axis = doc.newAxisDescriptor()
+    axis.tag = "styl"
+    axis.name = "Style"
+    doc.addAxis(axis)
+
```

Comment:
Add a comment here saying: this style axis spans a range of 0 to 1, but only has a source at style = 0, and doesn't have an explicit mapping. We want to make sure that the roundtrip still stores the information that the max is 1.